usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -80,7 +81,9 @@ def post(self, request):
         json_request = self.validate_request_data(request.data)
         self.is_subaward = json_request["subawards"]
         self.constants = GLOBAL_MAP["subaward"] if self.is_subaward else GLOBAL_MAP["award"]
-        self.filters = json_request["filters"]
+        self.filters = add_date_range_comparison_types(
+            json_request.get("filters"), self.is_subaward, gte_date_type="action_date", lte_date_type="date_signed"
```

Comment:
These fields are flipped, `date_signed` is the start of an award (earliest transaction's action_date), `action_date` is the latest transaction's action_date

Reply:
Ignore the above comment, the logic is correct since we want any award which intersects the range